### PR TITLE
Run xxx-taskrunner develop containerrs with hotswapping 

### PR DESCRIPTION
**Pull request by kbirk**
This volume mounts the local source code and runs both the latest python code and the springboot server with a continuous develop process.

### CODE REVIEW

File: packages/mira/Dockerfile [hunk 1]
```diff
@@ -16,32 +45,28 @@ WORKDIR /
...
-COPY --from=taskrunner_builder /taskrunner/build/libs/*.jar /taskrunner.jar
-
-# Copy the echo script for testing
-COPY ./packages/taskrunner/src/test/resources/echo.py /echo.py
+COPY --from=mira_taskrunner_builder /taskrunner/build/libs/*.jar /taskrunner.jar
 
 # Install Mira
-RUN echo "Cache bust so we can always pull latest MIRA" $(date -u)
 COPY ./packages/mira/mira-version.txt /miraVersion.txt
 RUN COMMIT_SHA=$(cat /miraVersion.txt) && \
```
Comment (dgauldie) [inline packages/mira/Dockerfile:58]: Quote this to prevent word splitting

File: packages/mira/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+RUN add-apt-repository ppa:deadsnakes/ppa
+RUN apt-get update && apt-get install -y --no-install-recommends \
+	python3.10 \
+	wget \
+	python3-pip && \
+	rm -rf /var/lib/apt/lists/*
+
+# Install Mira
+COPY ./packages/mira/mira-version.txt /miraVersion.txt
+RUN COMMIT_SHA=$(cat /miraVersion.txt) && \
```
Comment (dgauldie) [inline packages/mira/Dockerfile:21]: Quote this to prevent word splitting

File: packages/mira/Dockerfile [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+	echo "Using MIRA commit $COMMIT_SHA" && \
+	wget -O mira.tar.gz https://github.com/gyorilab/mira/archive/${COMMIT_SHA}.tar.gz && \
```
Comment (dgauldie) [inline packages/mira/Dockerfile:23]: add --progress=dot:giga to avoid excessively bloated build logs

File: packages/funman/Dockerfile [hunk 4]
```diff
@@ -1,5 +1,34 @@
...
+	python3.10 \
+	wget \
+	python3-pip && \
+	rm -rf /var/lib/apt/lists/*
+
+# Install funman-api
+COPY ./packages/funman/funman-version.txt /funmanVersion.txt
+RUN git clone https://github.com/DARPA-ASKEM/funman-api.git
+WORKDIR /funman-api
+RUN COMMIT_SHA=$(cat /funmanVersion.txt) && \
```
Comment (dgauldie) [inline packages/funman/Dockerfile:23]: Quote this to prevent word splitting

**Review by YohannParis — APPROVED** [review]

**Review by Tom-Szendrey — APPROVED** [review]

**Review by dgauldie — APPROVED** [review]

**Pull request merged**